apps/arweave/src/ar_data_sync.erl

```diff
@@ -544,7 +549,7 @@ init("default" = StoreID) ->
 				Free = proplists:get_value(free_memory, memsup:get_system_memory_data(),
 						2000000000),
 				Limit2 = min(1000, erlang:ceil(Free * 0.9 / 3 / 262144)),
-				Limit3 = Limit2 - Limit2 rem 100 + 100,
+				Limit3 = ar_util:ceil_int(Limit2, 100),
```

Comment:
is this making 1.1 -> 2 (ceil) or could math_min work better here?

Reply:
This makes 110 -> 200 - i.e. it rounds up to the nearest multiple of 100

Reply:
ahhh